pkg/ctl/implementation.go

```diff
@@ -355,6 +357,13 @@ func (impl *defaultVexCtlImplementation) Merge(
 				}
 			}
 
+			// If statement does not have a timestamp, cascade
+			// the timestamp down from the document.
+			// See https://github.com/chainguard-dev/vex/issues/49
+			if s.Timestamp == nil {
+				s.Timestamp = doc.Timestamp
+			}
```

Comment:
Question: It's not a guarantee in this part of the code that `doc.Timestamp` is not `nil`, is that right? I'm thinking we might want to require a non-nil `doc.Timestamp` before we apply it to the statement. If the document doesn't have a timestamp, and the statement doesn't have a timestamp, we know we'll be producing a corrupt VEX history, and should probably error out. WDYT?

Reply:
Sounds good to me! 
I've added a check and error before assigning it blindly

Reply:
Thanks!